modules/nf-core/gatk4/svannotate/tests/main.nf.test

```diff
@@ -0,0 +1,133 @@
+nextflow_process {
+
+    name "Test Process GATK4_SVANNOTATE"
+    script "../main.nf"
+    process "GATK4_SVANNOTATE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "manta"
+    tag "manta/germline"
+    tag "gatk4"
+    tag "gatk4/svannotate"
+
+    test("test_gatk4_svannotate") {
+        setup {
+            run("MANTA_GERMLINE") {
+                script "../../../manta/germline/main.nf"
+                process {
+                    """
+                    input[0] = Channel.of([ [ id:'test' ], file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_cram'], checkIfExists: true), file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_cram_crai'], checkIfExists: true), [], []])
+                    input[1] = Channel.value([ [ id:'test' ], file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)])
+                    input[2] = Channel.value([ [ id:'test' ], file(params.test_data['homo_sapiens']['genome']['genome_fasta_fai'], checkIfExists: true)])
+                    input[3] = []
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = MANTA_GERMLINE.out.diploid_sv_vcf.combine(MANTA_GERMLINE.out.diploid_sv_vcf_tbi, by:0).map({ meta, vcf, tbi -> [ meta, vcf, tbi, [] ]})
+                input[1] = []
+                input[2] = []
+                input[3] = []
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.annotated_vcf.get(0).get(1)).name,
+                    process.out.index.get(0).get(1),
+                    )
+                .match("test_gatk4_svannotate") }
+            )
+        }
+
```

Comment:
```suggestion
```

Reply:
I have been chasing these in all the tests I check, is this created by tools?

Reply:
Yes!